polkadot/transaction-pool/src/lib.rs

```diff
@@ -172,38 +169,38 @@ impl txpool::Scoring<VerifiedTransaction> for Scoring {
 }
 
 /// Readiness evaluator for polkadot transactions.
-pub struct Ready<'a, T: 'a + PolkadotApi> {
+pub struct Context<'a, T: 'a + PolkadotApi> {
 	at_block: T::CheckedBlockId,
 	api: &'a T,
 	known_nonces: HashMap<AccountId, ::primitives::Index>,
-	known_indexes: HashMap<AccountIndex, AccountId>,
+	known_indexes: Mutex<HashMap<AccountIndex, AccountId>>,
```

Comment:
nit: indices

Reply:
it's intentionally not, since that was the old name for `known_nonces` and i didn't want to risk copy'n'paste errors. the naming needs a rethink in general.